modules/oe_content_publication/config/post_updates/00004_create_fields/field.field.node.oe_publication.oe_departments.yml

```diff
@@ -0,0 +1,32 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_departments
+    - node.type.oe_publication
+  module:
+    - rdf_skos
+id: node.oe_publication.oe_departments
+field_name: oe_departments
+entity_type: node
+bundle: oe_publication
+label: 'Related department'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:skos_concept'
+  handler_settings:
+    target_bundles: null
+    sort:
+      field: _none
+    auto_create: false
+    concept_schemes: {  }
+    field:
+      field_name: oe_departments
+      entity_type: node
+      bundle: oe_publication
+      concept_schemes: {  }
```

Comment:
I found this diff with the one stored in the `/config/install` directory:

```diff
diff --git a/modules/oe_content_publication/config/post_updates/00004_create_fields/field.field.node.oe_publication.oe_departments.yml b/modules/oe_content_publication/config/post_updates/00004_create_fields/field.field.node.oe_publication.oe_departments.yml
index b36a7b1..6168361 100644
--- a/modules/oe_content_publication/config/post_updates/00004_create_fields/field.field.node.oe_publication.oe_departments.yml
+++ b/modules/oe_content_publication/config/post_updates/00004_create_fields/field.field.node.oe_publication.oe_departments.yml
@@ -23,10 +23,12 @@ settings:
     sort:
       field: _none
     auto_create: false
-    concept_schemes: {  }
+    concept_schemes:
+      - 'http://publications.europa.eu/resource/authority/corporate-body'
     field:
       field_name: oe_departments
       entity_type: node
       bundle: oe_publication
-      concept_schemes: {  }
+      concept_schemes:
+        - 'http://publications.europa.eu/resource/authority/corporate-body'
 field_type: skos_concept_entity_reference
```

Reply:
I updated the file.